Gems/ROS2/Code/Source/Spawner/ROS2SpawnerComponent.h

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include "ROS2/Spawner/SpawnerBus.h"
```

Comment:
The include should use angle brackets.
```suggestion
#include <ROS2/Spawner/SpawnerBus.h>
```